code/scripting/api/objs/missionhotkey.cpp

```diff
@@ -52,7 +54,25 @@ ADE_VIRTVAR(Type,
 		LuaError(L, "This property is read only.");
 	}
 
-	return ade_set_args(L, "i", current.getLine()->type);
+	switch (current.getLine()->type) {
+		case HotkeyLineType::NONE:
+			eh_idx = LE_HOTKEY_LINE_NONE;
+			break;
+		case HotkeyLineType::HEADING:
+			eh_idx = LE_HOTKEY_LINE_HEADING;
+			break;
+		case HotkeyLineType::WING:
+			eh_idx = LE_HOTKEY_LINE_WING;
+			break;
+		case HotkeyLineType::SHIP:
+			eh_idx = LE_HOTKEY_LINE_SHIP;
+			break;
+		case HotkeyLineType::SUBSHIP:
+			eh_idx = LE_HOTKEY_LINE_SUBSHIP;
+			break;
```

Comment:
Switch should have a default case per your own previous reviews just in case more hotkey enums are added in the future for whatever reason.